client/blocks/inline-help/admin-sections.js

```diff
@@ -0,0 +1,381 @@
+		description: translate( 'Export posts, pages and more from your site.' ),
+		link: `/settings/export/${ siteSlug }`,
+		synonyms: [ 'xml', 'images', 'migration', 'import', 'download' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Manage sharing and social media connections' ),
+		description: translate(
+			'You can customize your website by changing the footer credit in customizer.'
+		),
+		link: `/sharing/${ siteSlug }`,
+		synonyms: [ 'facebook', 'twitter', 'twitter', 'tumblr', 'eventbrite' ],
+		icon: 'share',
+	},
+	{
+		title: translate( 'Add sharing buttons to my site' ),
+		description: translate(
+			'Allow readers to easily share your posts with others by adding sharing buttons throughout your site.'
+		),
+		link: `/sharing/buttons/${ siteSlug }`,
+		synonyms: [ 'like', 'reblog' ],
+		icon: 'share',
+	},
+	{
+		title: translate( 'Install, manage, and search for site Plugins' ),
+		description: translate(
+			'You can customize your website by changing the footer credit in customizer.'
+		),
+		link: `/plugins/${ siteSlug }`,
+		synonyms: [ 'upload' ],
+		icon: 'plugins',
+	},
+	{
+		title: translate( 'Approve or delete comments' ),
+		link: `/comments/all/${ siteSlug }`,
+		synonyms: [ 'spam', 'discussion', 'moderation', 'moderate' ],
+		icon: 'chat',
+	},
+	{
+		title: translate( 'Manage how users can comment on my site' ),
+		link: `/settings/discussion/${ siteSlug }`,
+		synonyms: [ 'discussion', 'moderation', 'blacklist' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Manage post categories' ),
+		link: `/settings/writing/${ siteSlug }`,
+		synonyms: [ 'post', 'category' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Edit my site title, tagline, or logo' ),
+		link: `/settings/general/${ siteSlug }`,
+		synonyms: [ 'title', 'logo' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Set up a podcast' ),
+		link: `/settings/writing/${ siteSlug }`,
+		synonyms: [ 'podcast', 'radio', 'audio' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( "Change my site's privacy settings" ),
+		link: `/settings/general/${ siteSlug }`,
+		synonyms: [ 'privacy' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Manage SEO and traffic settings' ),
+		link: `/settings/traffic/${ siteSlug }`,
+		synonyms: [ 'analytics', 'related', 'sitemap' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Update my profile' ),
+		description: translate( 'Update your name, profile image, and about text.' ),
+		link: '/me',
+		synonyms: [ 'avatar' ],
+		icon: 'user',
+	},
+	{
+		title: translate( 'Update my username or email address' ),
+		link: '/me/account',
+		synonyms: [ 'user', 'account' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Change the dashboard color scheme' ),
+		link: '/me/account',
+		synonyms: [ 'theme' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( 'Switch the interface language' ),
+		description: translate(
+			'Update the language of the interface you see across WordPress.com as a whole.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **37** times:
`translate( 'This is the language of the interface you see across WordPress.com as a whole.' )` *ES Score: 7*